concepts/05 UI Components/TextArea/00 Getting Started with TextArea/10 Specify Value and Label.md

```diff
@@ -0,0 +1,87 @@
+To set an initial TextArea value, use the [value](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#value) property. You can also specify the [maxLength](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#maxLength) property to limit the number of characters users can enter. Note that this property only limits the number of characters for users. You can enter text that exceeds the maximum character length programmatically, but the number of characters displayed to users will still be limited by this property setting.
```

Comment:
If we're talking about setting the value programmatically (in code) I wouldn't use the verb "enter". 

```suggestion
To set an initial TextArea value, use the [value](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#value) property. You can also specify the [maxLength](/Documentation/ApiReference/UI_Components/dxTextArea/Configuration/#maxLength) property to limit the number of characters users can enter. Note that this property only limits the number of characters for users. Your code can specify text that exceeds the maximum character length, but the number of characters displayed to users will still be limited by this property setting.
```